operator/docs/user-guides/open-telemetry.md

```diff
@@ -0,0 +1,163 @@
+
+```yaml
+# [...]
+spec:
+  storage:
+    schemas:
+    - version: v13
+      effectiveDate: 2024-10-25
+```
+
+Once the `effectiveDate` has passed your `LokiStack` will be using the new schema configuration and is ready to store structured metadata.
+
+## Attribute Mapping
+
+Loki splits the configuration for mapping OTLP attributes to stream labels and structured metadata into two places:
+
+- `default_resource_attributes_as_index_labels` in the [`distributor` configuration](loki-docs-distributor-config)
+- `otlp_config` in the `limits_config` (see [Loki documentation](loki-docs-limits-config))
+
+By default, `default_resource_attributes_as_index_labels` provides a set of resource-attributes that are mapped to stream-labels on the Loki side.
+
+As the field in the distributor configuration is limited to resource-level attributes and can only produce stream-labels as an output, the `otlp_config` needs to be used to map scope or log level attributes to structured metadata.
+
+**Note:** All attributes that are not mapped to either a stream label or structured metadata will not be stored into Loki.
+
+### Operator Defaults
+
+When using the Loki Operator the default attribute mappings depend on the [tenancy mode]({{< ref "api.md#loki-grafana-com-v1-ModeType" >}}) used for the `LokiStack`:
+
+- `static` and `dynamic` use the Grafana defaults
+- `openshift-logging` uses OpenShift defaults
+
+### Custom Attribute Mapping
+
+All tenancy modes support customization of the attribute mapping configuration. This can be done globally (for all tenants) or on a per-tenant basis. When a custom attribute mapping configuration is defined, then the Grafana defaults are not used. If the default labels are desired as well, they need to be added to the custom configuration. See also the section about [Customizing OpenShift Defaults](#customizing-openshift-defaults) below.
+
+**Note:** A major difference between the Operator and Loki is how it handles inheritance. Loki by default only copies the attributes defined in the `default_resource_attributes_as_index_labels` setting to the tenants whereas the Operator will copy all global configuration into every tenant.
+
+The attribute mapping configuration in `LokiStack` is done through the limits configuration:
+
+```yaml
+# [...]
+spec:
+  limits:
+    global:
+      otlp: {} # Global OTLP Attribute Configuration
+    tenants:
+      example-tenant:
+        otlp: {} # OTLP Attribute Configuration for tenant "example-tenant"
+```
+
+Both global and per-tenant OTLP configuration can contain rules mapping attributes to stream-labels or structured-metadata. At least _one stream-label_ is needed for successfully saving a log entry to Loki storage, so the configuration should account for that.
```

Comment:
Since we don't mention "rules" anywhere else in the document 
```suggestion
Both global and per-tenant OTLP configurations can map attributes to stream-labels or structured-metadata. At least _one stream-label_ is needed for successfully saving a log entry to Loki storage, so the configuration should account for that.
```